### PR TITLE
Chore update sdks

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

(Provide a description of what this PR does and why it's needed.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -199,6 +199,10 @@ RUN chmod +x /usr/local/bin/hamster && \
 # Letsencrypt Permissions
 RUN mkdir -p /etc/letsencrypt/live/ && chmod -Rf 755 /etc/letsencrypt/live/
 
+RUN apk add git openssh-client
```
Comment (eldadfux) [inline Dockerfile:202]: Leftover.

File: Dockerfile [hunk 2]
```diff
@@ -199,6 +199,10 @@ RUN chmod +x /usr/local/bin/hamster && \
 # Letsencrypt Permissions
 RUN mkdir -p /etc/letsencrypt/live/ && chmod -Rf 755 /etc/letsencrypt/live/
 
+RUN apk add git openssh-client 
+RUN git config --global user.email "christyjacob4@gmail.com"
+RUN git config --global user.name "Christy Jacob"
```
Comment (Meldiron) [inline Dockerfile:204]: Is this needed?

File: composer.json [hunk 3]
```diff
@@ -86,7 +86,7 @@
         }
     ],
     "require-dev": {
-        "appwrite/sdk-generator": "0.34.*",
+        "appwrite/sdk-generator": "dev-master",
```
Comment (Meldiron) [inline composer.json:89]: Let's release version if possible

File: docker-compose.yml [hunk 4]
```diff
@@ -84,6 +84,7 @@ services:
       - ./public:/usr/src/code/public
       - ./src:/usr/src/code/src
       - ./dev:/usr/src/code/dev
+      - /Users/christyjacob/.ssh:/root/.ssh
```
Comment (Meldiron) [inline docker-compose.yml:87]: Leftover

File: Dockerfile (hunk 2 above)
Comment (christyjacob4) [reply Dockerfile]: removed

File: composer.json (hunk 3 above)
Comment (christyjacob4) [reply composer.json]: released and udpated

File: docker-compose.yml (hunk 4 above)
Comment (christyjacob4) [reply docker-compose.yml]: removed

File: Dockerfile (hunk 1 above)
Comment (christyjacob4) [reply Dockerfile]: removed 👍

File: docs/sdks/dart/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,20 @@
...
+* Added a new `label` function to the `Role` helper class
+* Update internal variable names to prevent name collision
+* Fix: content range header inconsistency in chunked uploads [#648](https://github.com/appwrite/sdk-generator/pull/648) 
+
+## 8.0.1
+
+* Added documentation comments
+* Added unit tests
+* Upgraded dependencies
+
```
Comment (stnguyen90) [inline docs/sdks/dart/CHANGELOG.md:17]: order is off

**Review by stnguyen90 — APPROVED** [review]

File: docs/sdks/dart/CHANGELOG.md (hunk 5 above)
Comment (christyjacob4) [reply docs/sdks/dart/CHANGELOG.md]: updated 👍

**Pull request merged**